torch/testing/_internal/common_nn.py

```diff
@@ -4903,7 +4930,7 @@ def test_cuda(self, test_case):
             raise unittest.SkipTest('Excluded from CUDA tests')
 
         cpu_input = self._get_input()
-        type_map = {'torch.DoubleTensor': torch.cuda.FloatTensor}
+        type_map = {'torch.DoubleTensor': torch.cuda.FloatTensor}  # type: ignore[attr-defined]
```

Comment:
`DoubleTensor` et al. are missing from type stubs on purpose, so this ignore is fine